app/controllers/posts_controller.rb

```diff
@@ -20,11 +21,11 @@ def create
     @new_post.user = current_user
 
     if @new_post.save
+      flash.now[:notice] = 'You have successfully created a new post.'
       @new_post.update_post_counter
-      flash[:notice] = 'You have successfully created a new post.'
-      redirect_to user_path(@new_post.user_id), notice: 'Post created successfully'
+      redirect_to user_path(@new_post.user_id)
     else
-      render :new, alert: 'An error occurred while creating the post'
+      render :new, alert: 'An error has occurred while creating the post'
     end
```

Comment:
- [ ] After I created a new post, I didn't receive a success message. Let's fix this. Hint below

```ruby
    if @new_post.save
      @new_post.update_post_counter
      redirect_to user_path(@new_post.user_id), notice: 'You have successfully created a new post.'
    else
      render :new, alert: 'An error has occurred while creating the post'
    end
```